PERSISTENT-WORKERS.md

```diff
@@ -74,6 +77,43 @@ log:
   max-rotations: 10
 ```
 
+### Resource management
+
+Persistent Worker supports resource management, [similarly to Kubernetes](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/), but in a slightly more simplified way.
+
+Resources are key-value pairs, where key represents an arbitrary resource name, and value is a floating-point number specifying how many of that resource the worker has.
+
+When scheduling tasks, Cirrus CI ensures that all the tasks the worker receives do not exceed the resources defined in it's configuration file, for example, with the following configuration:
+
+```yaml
+token: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+name: "mac-mini-1"
+
+resources:
+  tart-vms: 2
+```
+
+...a worker won't run more than 2 tasks simultaneously from the following `.cirrus.yml`:
+
+```yaml
+persistent_worker:
+  resources:
+    tart-vms: 1
```

Comment:
Maybe add `isolation` as well here? To be a bit more close to real life use case.